float64[3] test
int64[3] test2
